bin/cron/deliver_poste_messages

```diff
@@ -293,6 +294,7 @@ def main
         raise
       end
 
+      hashed_email = Digest::MD5.hexdigest(delivery[:contact_email])
```

Comment:
can't we just read `delivery[:hashed_email]`, set [here](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/poste.rb#L252)